doc/api/resources/submissions.rst

```diff
@@ -22,6 +22,7 @@ state                                 string                     The submission'
 abstract                              string                     The abstract, a short note of the submission's content
 description                           string                     The description, a more expansive description of the submission's content
 duration                              number                     The talk's duration in minutes, or ``null``
+available_count                       number                     How often the talk is available for scheduling
```

Comment:
I think this name is pretty misleading, ``slot_count`` would work better.

Reply:
done.
i like `slot_count`! 
my choice was a *i don't have a good name for this* - so i looked around what could fit from the already available namings..